### PR TITLE
issue #542 Update sanirize-html to version 2.6.1

### LINKED ISSUE #542: Latest version (2.5.1) of sanitize-html cuts extra word
**UPDATE** so far there is already version 2.5.2, but the problem still happen. 

Latest version (2.5.1) of sanitize-html cuts extra word:
```
AssertionError {
    actual: {
      replyType: 'plain',
      subject: 'tiny inline img plain',
      text: `Below␊
      [image: image.png]`,
    },
    expected: {
      replyType: 'plain',
      subject: 'tiny inline img plain',
      text: `Below␊
      [image: image.png]␊
      Above`,
    },
    message: `expected { text: 'Below\\n[image: image.png]',␊
      replyType: 'plain',␊
      subject: 'tiny inline img plain' } to deeply equal { text: 'Below\\n[image: image.png]\\nAbove',␊
      replyType: 'plain',␊
      subject: 'tiny inline img plain' }`,
    operator: 'deepStrictEqual',
    showDiff: true,
  }
```
`Above` is cut away
So I am going to revert and restore the fix.
It can be separate issue to upgrade it and fix this.

_Originally posted by @IvanPizhenko in https://github.com/FlowCrypt/flowcrypt-ios/pull/541#issuecomment-926958831_

### PR DESCRIPTION
**Pull request by IvanPizhenko**
This PR updates sanitize-html to latest version 2.6.1

close #542

----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities
- [ ] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

File: Core/source/platform/util.ts [hunk 1]
```diff
@@ -30,3 +30,9 @@ export const setGlobals = () => {
 export const iso2022jpToUtf = (content: Buf) => {
   return dereq_encoding_japanese.convert(content, { to: 'UTF8', from: 'JIS', type: 'string' });
 }
+
+export const debugPrintArray = (title: string, arr: any[]) => {
```
Comment (sosnovsky) [inline Core/source/platform/util.ts:34]: Is it used anywhere?

**Review by sosnovsky — COMMENTED** [review]
Looks good, but tests are failing

File: Core/source/platform/util.ts (hunk 1 above)
Comment (tomholub) [reply Core/source/platform/util.ts:34]: I wrote this. No longer needed, I think.

**Comment by IvanPizhenko** [comment]
> Looks good, but tests are failing

Yes, tests are failing, but I don't have any ideas what can be wrong. Locally all tests written in Typescript are passing.
The rest I can't check as long as I'm on Linux, not Mac.

 @tomholub I suggest to merge now as is and let others fix tests.

**Comment by tomholub** [comment]
I don't want to merge until it passes all CI tests. @sosnovsky could you please take a look?

**Comment by sosnovsky** [comment]
> I don't want to merge until it passes all CI tests. @sosnovsky could you please take a look?

Yes, I'll check it

**Review by sosnovsky — APPROVED** [review]

**Pull request merged**